tap-snapshots/test-lib-view.js-TAP.test.js

```diff
@@ -0,0 +1,290 @@
+
+
+[4m[1m[32mgreen[39m@[32mundefined[39m[22m[24m | [32mACME[39m | deps: [36m2[39m | versions: [33m2[39m
+
+[1m[31mDEPRECATED[39m[22m ⚠️  - true
+
+keywords:[33mcolors[39m, [33mgreen[39m, [33mcrayola[39m
+
+bin:[33mgreen[39m
+
+dist
+.tarball:[36mundefined[39m
+.shasum:[33mundefined[39m
+.integrity:[33m---[39m
+.unpackedSize:[33m1[39m B
+
+dependencies:
+[33mred[39m: [object Object]
+[33myellow[39m: [object Object]
+
+dist-tags:
+[1m[32m1.0.1[39m[22m: [object Object]
+`
+
+exports[`test/lib/view.js TAP should log package info package with maintainers info as object > must match snapshot 1`] = `
+
+
+[4m[1m[32mpink[39m@[32mundefined[39m[22m[24m | [1m[31mProprietary[39m[22m | deps: [32mnone[39m | versions: [33m2[39m
+
+dist
+.tarball:[36mundefined[39m
+.shasum:[33mundefined[39m
+.integrity:[33m---[39m
+.unpackedSize:[33m1[39m B
+
+dist-tags:
+[1m[32m1.0.1[39m[22m: [object Object]
+`
+
+exports[`test/lib/view.js TAP should log package info package with more than 25 deps > must match snapshot 1`] = `
+
+
+[4m[1m[32mblack[39m@[32mundefined[39m[22m[24m | [1m[31mProprietary[39m[22m | deps: [36m25[39m | versions: [33m2[39m
+
+dist
+.tarball:[36mundefined[39m
+.shasum:[33mundefined[39m
+.integrity:[33m---[39m
+.unpackedSize:[33m1[39m B
+
+dependencies:
+[33m0[39m: [object Object]
+[33m10[39m: [object Object]
+[33m11[39m: [object Object]
+[33m12[39m: [object Object]
+[33m13[39m: [object Object]
+[33m14[39m: [object Object]
+[33m15[39m: [object Object]
+[33m16[39m: [object Object]
+[33m17[39m: [object Object]
+[33m18[39m: [object Object]
+[33m19[39m: [object Object]
+[33m1[39m: [object Object]
+[33m20[39m: [object Object]
+[33m21[39m: [object Object]
+[33m22[39m: [object Object]
+[33m23[39m: [object Object]
+[33m2[39m: [object Object]
+[33m3[39m: [object Object]
+[33m4[39m: [object Object]
+[33m5[39m: [object Object]
+[33m6[39m: [object Object]
+[33m7[39m: [object Object]
+[33m8[39m: [object Object]
+[33m9[39m: [object Object]
+(...and 1 more.)
+
+dist-tags:
+[1m[32m1.0.1[39m[22m: [object Object]
+`
+
+exports[`test/lib/view.js TAP should log package info package with no modified time > must match snapshot 1`] = `
+
+
+[4m[1m[32mcyan[39m@[32mundefined[39m[22m[24m | [1m[31mProprietary[39m[22m | deps: [32mnone[39m | versions: [33m2[39m
+
+dist
+.tarball:[36mundefined[39m
+.shasum:[33mundefined[39m
+
+dist-tags:
+
+
+published by [33mclaudia[39m <[36mclaudia@cyan.com[39m>
+`
+
+exports[`test/lib/view.js TAP should log package info package with no repo or homepage > must match snapshot 1`] = `
+
+
+[4m[1m[32mblue[39m@[32mundefined[39m[22m[24m | [1m[31mProprietary[39m[22m | deps: [32mnone[39m | versions: [33m2[39m
+
+dist
+.tarball:[36mundefined[39m
+.shasum:[33mundefined[39m
```

Comment:
there are quite a few `undefined` all over the snapshots, are these expected due to the nature of the mocked data?